data/gui/window/community_dialog.cfg

```diff
@@ -0,0 +1,184 @@
+[window]
+	id = "community_dialog"
+	description = "Dialog for links to different places related to Wesnoth's community"
```

Comment:
Should this be marked for translation?

Reply:
This is an internal description not used by the game, so it doesn't really need to be translatable. Its purpose is probably for someone making a custom skin, I guess?

Reply:
Looks like it's mandatory and translatable. If that still means it's not actually visible anywhere that's really confusing. Maybe it either should be visible somewhere or removed. For an internal description a comment seems clearer.

I don't know what the custom skin comment means.

Reply:
That's right, the description key is required, but not user visible. It almost behaves like an internal comment to clarify the dialogs purpose.
> I don't know what the custom skin comment means.
Probably something related to ThemeWML?

Reply:
Perhaps the description key was meant to be used somewhere in the UI, but never implemented?

Reply:
Seems likely. I've never known what it was for, so I just add/updated it for any dialogs I added in case it was used for something.

Reply:
> I don't know what the custom skin comment means.

A custom GUI2 skin. The GUI2 system has a little-known system that lets you create themes which completely replace every dialog if you want to.